Changelog.md

```diff
@@ -10,6 +10,7 @@ Compiler Features:
  * NatSpec: Add support for NatSpec documentation in ``struct`` definitions.
  * NatSpec: Include NatSpec from events that are emitted by a contract but defined outside of it in userdoc and devdoc output.
  * Optimizer: Re-implement simplified version of UnusedAssignEliminator and UnusedStoreEliminator. It can correctly remove some unused assignments in deeply nested loops that were ignored by the old version.
+ * Parser: Introduce ``pragma experimental solidity``, which will be used to enable experimental features in the future, such as the standard library, new parsing behaviour, etc.
```

Comment:
Shouldn't we also add a section in the docs about this?

Reply:
At the moment, the pragma doesn't really do anything (aside from spewing an error here an there), so I'd say now. Once we start implementing features to go behind the pragma, i.e. postfix types, stdlib, etc., then it will make sense to add docs (perhaps even a special page with all experimental features).